falco.yaml

```diff
@@ -63,6 +63,26 @@ priority: debug
 # buffered. Defaults to false
 buffered_outputs: false
 
+# Falco uses a shared buffer between the kernel and userspace to pass
+# system call informaation. When falco detects that this buffer is
+# full and system calls have been dropped, it can take one or more of
+# the following actions:
+#   - "ignore": do nothing. If an empty list is provided, ignore is assumed.
+#   - "log": log a CRITICAL message noting that the buffer was full.
+#   - "alert": emit a falco alert noting that the buffer was full.
+#   - "exit": exit falco with a non-zero rc.
+#
+# The rate at which overflow log/alert messages are emitted is
+# governed by a token bucket. The rate corresponds to one message
+# every 30 seconds with a burst of 10 messages.
+
+kernel_buffer_overflow:
```

Comment:
For all occurrences (code/comments/logs) I would suggest using the term "ring buffer", maybe even "sysdig ring buffer", instead of "kernel buffer".

Reply:
New name: "syscall event drops"